scripts/just.config.ts

```diff
@@ -91,7 +91,7 @@ export function preset() {
     const args = getJustArgv();
 
     return args.commonjs
-      ? 'ts:commonjs-only'
+      ? series('ts:commonjs-only')
```

Comment:
returning string didn't work -> this was broken (never invoked if user specified `--commonjs`